pkg/webhook/stack/validatingwebhook_test.go

```diff
@@ -225,6 +225,28 @@ func TestValidatingWebhook9(t *testing.T) {
 		t.Fatal("Validation failed. A message was expected: ", msg)
 	}
 
+	if err == nil {
+		t.Fatal("Validation failed. An error was expected: ", err)
+	}
+}
+
+
+// Spec.Versions not semver
+func TestValidatingWebhook10(t *testing.T) {
+	newStack := validatingStack.DeepCopy()
+	newStack.Spec.Versions[0].Version = "1.0"
```

Comment:
Your test case illustrates perfectly how we can educate someone who really only understands semver a little bit, what they need to do to be compliant here.  1.0 should be 1.0.0.